spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -17,9 +17,20 @@ include:
   - gnss.yaml
 definitions:
   - MSG_ED25519_SIGNATURE:
-      id: 0x0C01
+      id: 0x0C03
```

Comment:
Don't we need to retain the old definition and message number as a `_DEP` message?

Reply:
Yes